### PR TITLE
Bloodhoof Runespear Artifact

### PR DESCRIPTION
**Pull request by arithon**
## Changelog:
- Added the Runespear of the Bloodhoof Clan.
**_Note:_** Image shall be reworked, once better image comes up.

### CODE REVIEW

File: common/artifacts/000_wc_artifacts.txt [hunk 1]
```diff
@@ -356,6 +356,31 @@ atiesh = {
 	picture = "GFX_inv_staff_medivh"
 	slot = weapon
 }
+bloodhoof_runespear = {
+	quality = 5
+	flags = { combat_weapon lance }
```
Comment (MemeWizard) [inline common/artifacts/000_wc_artifacts.txt]: 🤔 
```suggestion
	flags = { combat_weapon spear }
```

Comment (arithon) [reply common/artifacts/000_wc_artifacts.txt]: I hope, I'll learn this at one point 😅

Comment (arithon) [reply common/artifacts/000_wc_artifacts.txt]: done

File: common/artifacts/000_wc_artifacts.txt [hunk 2]
```diff
@@ -356,6 +356,31 @@ atiesh = {
 	picture = "GFX_inv_staff_medivh"
 	slot = weapon
 }
+bloodhoof_runespear = {
+	quality = 5
+	flags = { combat_weapon spear }
+
+	active = {
+		is_adult = yes
+		is_physical_class_trigger = yes
```
Comment (zumbak04) [inline common/artifacts/000_wc_artifacts.txt]: Paladins and character without class can't equip it. Is it ok?
```suggestion
		OR = {
			NOT = { has_class_trait_trigger = yes }
			is_physical_class_trigger = yes
			is_combined_class_trigger = yes
		}
```

Comment (arithon) [reply common/artifacts/000_wc_artifacts.txt]: done

**Pull request merged**